tests/integration/test_rex.py

```diff
@@ -7,6 +7,7 @@
 from pandas.testing import assert_frame_equal
 
 
+@pytest.mark.xfail
```

Comment:
We should probably give a reason here for clarity moving forward:

```suggestion
@pytest.mark.xfail(
    reason="Bumping to Calcite 1.29.0 to address CVE-2021-44228 caused a stack overflow in this test"
)
```